applications/serial_lte_modem/prj.conf

```diff
@@ -64,6 +64,7 @@ CONFIG_IMG_MANAGER=y
 CONFIG_FLASH=y
 CONFIG_FLASH_PAGE_LAYOUT=y
 CONFIG_FLASH_MAP=y
+CONFIG_STREAM_FLASH=y
 CONFIG_MPU_ALLOW_FLASH_WRITE=y
 CONFIG_IMG_ERASE_PROGRESSIVELY=y
 
```

Comment:
Please add CONFIG_SLM_TFTPC=n for now
Will update SLM TFTP client with a PR later

Reply:
okay, disabled SLM_TFTPC.